src/components/question_creation/question-types/coding-question.tsx

```
@@ -0,0 +1,604 @@
+            <CardHeader>
+              <CardTitle>Code Preview</CardTitle>
+            </CardHeader>
+            <CardContent className="space-y-6">
+              {/* Question Content Preview */}
+              <div>
+                <Label className="text-lg font-semibold">Question</Label>
+                <div className="mt-2 p-4 border rounded-lg bg-muted/50">
+                  {question ? (
+                    <div
+                      className="prose prose-sm max-w-none"
+                      dangerouslySetInnerHTML={{ __html: question }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)